model.py

```diff
@@ -339,6 +339,12 @@ def set_alias_id_mapping(self, alias, id):
             'player_alias': alias,
             'player_id': id
         })
+
+    def delete_alias_id_mapping(self, alias):
+        for mapping in self.alias_to_id_map:
+            if mapping['player_alias'] == alias:
+                self.alias_to_id_map.remove(mapping)
+                return mapping
```

Comment:
Can you explicitly `return None` outside of the loop in case we don't find a match?